specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -1850,62 +2174,6 @@
       "description": "The resource model definition.",
       "x-ms-azure-resource": true
     },
```

Comment:
You really shouldn't be defining TrackedResource yourself, you should just inherit the one from common-types instead.